packages/features/bookings/tests/setenv.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+# scripts/setenv.sh
+
+# Export env vars
+export $(grep -v '^#' .env | xargs)
+export DATABASE_URL="postgres://postgres:postgres@localhost:5432/postgres"
```

Comment:
This script imports all .env variables but we want to overwrite the `DATABASE_URL` to point to our testing DB.